geth/txqueue/utils.go

```diff
@@ -0,0 +1,28 @@
+package txqueue
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/signal"
+)
+
+var ErrTxQueueRunFailure = errors.New("error running transaction queue")
```

Comment:
Hm, it looks like this is not used anywhere except for a `fmt.Errorf()` below.

Reply:
Correct. Idea was to introduce as little changes as possible in this PR. Just extracting code into separate package.